src/app/public/modules/tabs/tab-dropdown.component.html

```diff
@@ -9,22 +9,18 @@
         <div
           class="sky-tab-dropdown-item"
           [ngClass]="{ 'sky-tab-dropdown-item-selected': tab.active }">
-          <button
-            class="sky-btn sky-tab-dropdown-item-btn sky-padding-even-default"
-            type="button"
+          <sky-tab-button
```

Comment:
Instead of using something custom for the dropdown, I chose to repurpose the tab button component so that both modes have the same underlying HTML elements and selectors.